.github/pull_request_template.md

```diff
@@ -1,11 +1,16 @@
 <!--
 Thanks for contributing to Backpack :pray:
+
 Please include a description of the changes you are introducing and some screenshots if appropriate.
+
+Please ensure your pull request title is clear as it will be used to generate the changelog.
+
+Add `major`, `minor` or `patch` label depending on the change according to [semver](semver.org) or `skip-changelog` if the change shouldn't be added to the changelog (e.g. a change to a test or documentation)
 -->
 
 Remember to include the following changes:
 
-- [ ] `UNRELEASED.md` (See [How to write a good changelog entry](https://github.com/Skyscanner/backpack/blob/main/CHANGELOG_FORMAT.md))
 - [ ] `README.md`
```

Comment:
Changing the README.md should be an exception right? Component README.md files are the ones to look out for so perhaps drop it?

Reply:
True! It's mainly for if its a brand new component, so can clarify it?